algorithms/kernel/pca/transform/pca_transform_container.h

```diff
@@ -28,6 +28,9 @@
 
 #include "pca_transform_batch.h"
 #include "pca_transform_kernel.h"
+#include "oneapi/internal/utils.h"
+#include "oneapi/pca_transform_dense_default_batch_oneapi.h"
+#include "oneapi/pca_transform_dense_default_batch_oneapi_impl.i"
```

Comment:
You have not to include the **.i** files into other headers. The only place when you should do it is the **.cpp** file with the explicit instantiation of your `TransformKernelOneAPI`.